packages/constants/src/mcf.ts

```diff
@@ -298,3 +298,12 @@ export const COMPANY_ADDRESS_PURPOSES: ICompanyAddressPurpose[] = [
   {id: COMPANY_ADDRESS_PURPOSE.OPERATING, purpose: 'operating'},
   {id: COMPANY_ADDRESS_PURPOSE.CORRESPONDENCE, purpose: 'correspondence'},
 ];
+
+/*************************************
+ * Job Screening Question Response
+ *************************************/
+
+ export enum SCREEN_QNS_RESPONSE {
+  YES = 'yes',
```

Comment:
:thinking: will it be more useful if its a list of `{label: string; value: string}` instead of enum. since we need a consistent display of the label and value saved

Reply:
so to clarify our label: 'Yes', value: 'yes' ? 🤔

Reply:
haha probably. seems like the case

Reply:
yes i see the need to add label since we are displaying them in a few places. Updated this!

Reply:
hmmm. i am trying to see how we can make this more meaning full
some ideas, 3 examples below. Can come up with more or an hybrid of them
```ts
 export enum SCREEN_QUESTION_RESPONSE {
  YES
  NO
}
export interface IScreeningQuestionResponse {
  label: string;
  value: string;
}
export const SCREENING_QUESTION_RESPONSES: {[key: SCREEN_QNS_RESPONSE]: IScreeningQuestionResponse} = {
  [SCREEN_QUESTION_RESPONSE.YES]: {label: 'Yes' , value: 'yes'},
  [SCREEN_QUESTION_RESPONSE.NO]: {label: 'No', value: 'no'},
};

// usage
const yesValue = SCREENING_QUESTION_RESPONSES[SCREEN_QUESTION_RESPONSE.YES].value;
const noLabel = SCREENING_QUESTION_RESPONSES[SCREEN_QUESTION_RESPONSE.NO].label;
```

```ts
 export enum SCREEN_QUESTION_RESPONSE {
  YES
  NO
}
export interface IScreeningQuestionResponse {
  id: SCREEN_QUESTION_RESPONSE;
  label: string;
  value: string;
}
export const SCREENING_QUESTION_RESPONSES: IScreeningQuestionResponse[] = [
  {id: SCREEN_QUESTION_RESPONSE.YES, label: 'Yes' , value: 'yes'},
  {id: SCREEN_QUESTION_RESPONSE.NO, label: 'No', value: 'no'},
];

// usage
const yesValue = SCREENING_QUESTION_RESPONSES.find(({id}) => id === SCREEN_QUESTION_RESPONSE.YES).value; // problem with find is that it might end up with undefined
const noLabel = SCREENING_QUESTION_RESPONSES.find(({id}) => id === SCREEN_QUESTION_RESPONSE.NO).label;
```

```ts
 export enum SCREEN_QUESTION_RESPONSE {
  YES
  NO
}
export interface IScreeningQuestionResponse {
  label: string;
  value: string;
}
export const getScreeningQuestionResponse(enum: SCREEN_QUESTION_RESPONSE): IScreeningQuestionResponse => {
  switch (enum) {
        case SCREEN_QUESTION_RESPONSE.YES:
            return {label: 'Yes' , value: 'yes'};
        case SCREEN_QUESTION_RESPONSE.NO:
            return {label: 'No', value: 'no'};
    }
} 

// usage
const yesValue = getScreeningQuestionResponse(SCREEN_QUESTION_RESPONSE.YES).value;
const noLabel = getScreeningQuestionResponse(SCREEN_QUESTION_RESPONSE.NO).label;
```

Reply:
After digesting them, and since it can only be a `yes or no` values, i think there is no need to keep an `id` param. (correct me if there is usefulness) which is why I chose the switch case will just return the `label+value` object.